packages/compass-e2e-tests/tests/auto-connect.test.ts

```diff
@@ -179,6 +176,10 @@ describe('Automatically connecting from the command line', function () {
         { connectionStatus: 'failure' }
       );
       expect(error).to.include('Authentication failed');
+      if (TEST_MULTIPLE_CONNECTIONS) {
```

Comment:
In single connection world you land on the connection form, so there's no else necessary.